### PR TITLE
fix: flaky digest test

### PR DESCRIPTION
**Pull request by capJavert**
- now the dates should be faked and static no matter what is the actual time of the test, tried to run locally with changed time
- gonna run it also on CI throughout the day to see if its good now

### CODE REVIEW

File: __tests__/cron/personalizedDigest.ts [hunk 1]
```diff
@@ -38,32 +39,35 @@ jest.mock('../../src/common/personalizedDigest', () => ({
...
-      .setSystemTime(setDay(currentDate, 2));
+      .setSystemTime(fakeDate);
+
+    digestTime = format(new Date(), 'yyyy-MM-dd HH:mm:ss xxx');
+
+    return {
+      fakePreferredDay,
+      fakePreferredHour: preferredHour,
+    };
+  };
```
Comment (capJavert) [inline __tests__/cron/personalizedDigest.ts:64]: This now fakes jest time and cron time (in db queries)

**Review by omBratteng — APPROVED** [review]
🤷‍♂️

**Comment by capJavert** [comment]
I guess we will see 😆

**Comment by capJavert** [comment]
Looks like it works 🤞

**Pull request merged**